src/main/java/teammates/ui/controller/InstructorFeedbackRemindParticularStudentsPageAction.java

```diff
@@ -25,8 +25,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         FeedbackSessionResponseStatus fsResponseStatus =
                 logic.getFeedbackSessionResponseStatus(feedbackSessionName, courseId);
 
-        InstructorFeedbackRemindParticularStudentsPageData data =
-                new InstructorFeedbackRemindParticularStudentsPageData(account, sessionToken, fsResponseStatus,
+        InstructorFeedbackAjaxStudentsListPageData data =
+                new InstructorFeedbackAjaxStudentsListPageData(account, sessionToken, fsResponseStatus,
                                                                        courseId, feedbackSessionName);
```

Comment:
8-space indentation. Constructor call can be on the first line, all the parameters can be on the second.